### PR TITLE
Add deprecation message to README

### PR DESCRIPTION
**Pull request by elfosardo**
Also add --no-stream to tripleo repos to fix test-integration job.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,9 @@
-# ironic-inspector-image
+#  This repository is deprecated and not longer maintained
```
Comment (bfournie) [inline README.md]: Should be "...no longer maintained"

Comment (elfosardo) [reply README.md]: copy-paste failure! \o/

**Review by bfournie — APPROVED** [review]

**Comment by elfosardo** [comment]
@hardys @stbenjam @maelk @dtantsur the integration won't pass because of the "OS is not the Stream version" issue.
Considering that we're not accepting changes to the repo anymore, could we maybe force merge this?

**Comment by dtantsur** [comment]
I don't think I can force-merge stuff. Fixing it is trivial (just add `--no-stream` to `tripleo-repos`), maybe just do it here?

**Review by dtantsur — APPROVED** [review]
/lgtm

**Pull request merged**